addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,14 +2,16 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
+        <div t-attf-class="o_notification toast show border-{{props.type}} overflow-hidden mb-2"
```

Comment:
Bootstrap `.toast` component includes a margin-bottom of `1rem` for every toast but the last. Is it really necessary to override it by `mb-2`? Is it really too much?

Reply:
yes imo it's too much.

Reply:
@xlu-odoo You have the last word.